### PR TITLE
Fixed: UDF execution

### PR DESCRIPTION
**Pull request by agafox**

### CODE REVIEW

File: src/main/kotlin/io/sip3/salto/ce/sip/SipMessageHandler.kt [hunk 1]
```diff
@@ -144,17 +151,25 @@ open class SipMessageHandler : AbstractVerticle() {
...
-                put("attributes", packet.attributes)
                 put("payload", message.headersMap())
+                put("attributes", mutableMapOf<String, Any>())
             }
 
-            vertx.eventBus().request<Boolean>(Routes.sip_message_udf, udf, USE_LOCAL_CODEC) { asr ->
-                if (asr.failed()) {
-                    logger.info("SipMessageHandler `callUserDefinedFunction()` failed.", asr.cause())
+            GlobalScope.launch(vertx.dispatcher()) {
+                val result = withTimeoutOrNull(executeUdfTimeout) {
```
Comment (sip3leo) [inline src/main/kotlin/io/sip3/salto/ce/sip/SipMessageHandler.kt]: What do you think about using `*Await` functions? 
```kotlin
val result = withTimeoutOrNull(executeUdfTimeout) {
    vertx.eventBus().requestAwait<Boolean>(Routes.sip_message_udf, udf, USE_LOCAL_CODEC)
}
```
For me it's better to read and understand.

File: src/main/kotlin/io/sip3/salto/ce/sip/SipMessageHandler.kt [hunk 2]
```diff
@@ -65,15 +72,15 @@ open class SipMessageHandler : AbstractVerticle() {
...
             config.getJsonObject("sip")?.getJsonObject("message")?.getJsonArray("exclusions")?.let {
                 exclusions = it.map(Any::toString).toSet()
             }
-
-            config.getJsonObject("sip")?.getJsonObject("message")?.getLong("check-udf-period")?.let {
+            config.getJsonObject("udf")?.getLong("check-period")?.let {
                 checkUdfPeriod = it
             }
-
+            config.getJsonObject("udf")?.getLong("execute-period")?.let {
```
Comment (sip3leo) [inline src/main/kotlin/io/sip3/salto/ce/sip/SipMessageHandler.kt]: What is a final name for this parameter: `execute-timeout` or `execute-period`?

Comment (agafox) [reply src/main/kotlin/io/sip3/salto/ce/sip/SipMessageHandler.kt]: fixed, copy-paste typo

File: src/main/kotlin/io/sip3/salto/ce/sip/SipMessageHandler.kt (hunk 1 above)
Comment (agafox) [reply src/main/kotlin/io/sip3/salto/ce/sip/SipMessageHandler.kt]: agree

**Pull request merged**